src/python/pants/notes/1.30.x.rst

```diff
@@ -0,0 +1,205 @@
+1.30.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.30.x`` ``stable`` series.
+
+See https://pants.readme.io/v1.29/docs/release-notes-1-30 for an overview of the changes in this release.
+
+1.30.0dev0 (6/05/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Remove deprecated v1 functionality (#9996)
```

Comment:
Hm, even though this is the title, it's not very descriptive. Maybe rename to "Remove deprecated ExternalTool code". 

It is rewriting the past, but I argue that's the lesser sin.